src/components/__tests__/UserForm.test.tsx

```diff
@@ -5,7 +5,7 @@ describe('Input form', () => {
   test('input should initially be empty', () => {
     render(<UserForm />);
 
-    const userInput = screen.getByRole('textbox');
+    const userInput = screen.getByRole<HTMLInputElement>('textbox');
```

Comment:
Do we need to specify the type here?

Reply:
without it I get a red line under value in the next line:
[ expect(userInput.value).toBe('');](url)